third-party/chpl-venv/Makefile

```diff
@@ -32,22 +54,10 @@ check-exes: check-python
 $(CHPL_VENV_INSTALL_DIR):
 	mkdir -p $@
 
-# Install pip via get-pip.py to ensure we have an updated version
-$(PIP): $(CHPL_VENV_INSTALL_DIR)
-	@if [ -z "$$CHPL_PIP" ]; then \
-	  echo "Installing local copy of pip with get-pip.py from $(GETPIP)"; \
-	  (cd install && curl -O $(GETPIP) || \
-	  { echo "Failed on command: curl -O $(GETPIP)."; \
-	    echo "Try setting CHPL_PIP to $$(which pip) and trying again"; \
-	    exit 1; } && \
-	  $(GET_PIP_COMMAND) ) \
-	fi
-
 # Install virtualenv program.
-$(CHPL_VENV_VIRTUALENV): check-exes $(PIP)
-	export PYTHONPATH="$$CHPL_PYTHONPATH:$(PIPLIBS):$$PYTHONPATH" && \
-	$(PYTHON) $(PIP) install -U --force-reinstall --ignore-installed $(LOCAL_PIP_FLAGS) \
-	 --prefix=$(CHPL_VENV_INSTALL_DIR) $(CHPL_PIP_INSTALL_PARAMS) $(shell cat virtualenv.txt)
+$(CHPL_VENV_VIRTUALENV): check-exes
+	$(PIP) install -U --force-reinstall --ignore-installed $(LOCAL_PIP_FLAGS) \
```

Comment:
You've removed the PIP definition but you still use it here, is that okay?  Or are you getting it from another Makefile?

Reply:
Nevermind, I see it in the Makefile.include file